src/stores/RoomListStore.tsx

```diff
@@ -1,28 +1,28 @@
 import { makeAutoObservable } from "mobx";
 import { RootStore } from "@/stores/RootStore";
-import { RoomService } from "@/service/room.service";
+import roomService, { RoomService } from "@/service/room.service";
 import { RoomOverview } from "@/models/room/RoomOverview";
 import React from "react";
-import roomService from "@/service/room.service";
 
+//TODO(건우) 값을 임시로 할당하여 수정 필요
 export class Room {
-  //TODO(건우) 값을 임시로 할당하여 수정 필요
-  readonly id: string = "";
-  readonly masterId: string = "test";
-  readonly title: string = "test";
-  readonly thumbnail?: string = "test";
-  readonly password?: string = "test";
-  readonly timer: number = 2000;
-  readonly shortBreak: number = 15;
-  readonly longBreak: number = 20;
-  readonly longBreakInterval: number = 3;
-  readonly expiredAt: string = "2021-08-21T12:30:00.000Z";
-  readonly tags: string[] = [];
-
-  constructor() {
+  constructor(
+    readonly id: string = "",
+    readonly masterId: string = "test",
+    readonly title: string = "",
+    readonly thumbnail: string | undefined = undefined,
+    readonly password: string | undefined = undefined,
+    readonly timer: number = 30,
+    readonly shortBreak: number = 10,
+    readonly longBreak: number = 20,
+    readonly longBreakInterval: number = 3,
+    readonly expiredAt: string = "2021-08-21T12:30:00.000Z",
```

Comment:
지금 올린 PR과는 상관없는데여.. Date 타입으로 했어야할듯 하네요.
```suggestion
    readonly expiredAt: Date = new Date(),
```

Reply:
네 알겠습니다 !